shared/unattended/Fedora-22.ks

```diff
@@ -0,0 +1,36 @@
+install
+KVM_TEST_MEDIUM
+text
+reboot
+lang en_US
+keyboard us
+network --bootproto dhcp --hostname atest-guest
+rootpw 123456
+firewall --enabled --ssh
+selinux --enforcing
+timezone --utc America/New_York
+firstboot --disable
+bootloader --location=mbr --append="console=tty0 console=ttyS0,115200"
+zerombr
+poweroff
+KVM_TEST_LOGGING
+
+clearpart --all --initlabel
+autopart
+
+%packages
+@standard
```

Comment:
The `@development-tools` are missing from `Fedora-21`, is it intentional? Isn't it needed by some tests? (like `autotest` tests which need `gcc`?)